galaxy_api/auth/models.py

```diff
@@ -1,5 +1,34 @@
+from django.db import models
 from django.contrib.auth.models import AbstractUser
 
+from galaxy_api.common import models as common_models
+
+
+__all__ = (
+    'User',
+    'Tenant',
+)
+
 
 class User(AbstractUser):
+    """Custom user model."""
     pass
+
+
+class Tenant(common_models.TimestampsMixin):
```

Comment:
Slightly concerned about the Tenant name here. I usually think of tenant in the sense of multitenacy which feels like a broader concept.  That said, I don't have suggestions for a better name at the moment.

Reply:
Yep, I was struggling with naming either. Haven't found any better name so far.